l5kit/l5kit/simulation/unroll.py

```diff
@@ -105,12 +105,33 @@ def build_trajectory_states(frames: np.ndarray) -> torch.Tensor:
         return trajectory_states
 
 
+class SimulationOutput(SimulationOutputCLE):
```

Comment:
I think this will break some visualisation utils (e.g. `simulation_out_to_visualizer_scene`)

Reply:
The refactored SimulationOutput class has the same attributes as the original SimulationOutput class.  The tests are running fine. Would you suggest to check something regarding breaking visualization utils? 

Christian suggested to make SimulationOutput inherent from SimulationOutputCLE so that close-loop evaluator-specific components are kept separated from visualization components. Let me know if there is a better way to do this.